src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import './App.css';
+import Mission from './Components/Mission/Mission';
+import Nav from './Components/Nav';
+import Rockets from './Components/Rockets/Rocket';
+import Profile from './Components/Profile/Profile';
+
+function App() {
+  return (
+    <div className="App">
+      <Nav />
+      <Routes>
+        <Route exact path="/" element={<Rockets />} />
+        <Route path="/missions" element={<Mission />} />
+        <Route path="/profile" element={<Profile />} />
+      </Routes>
```

Comment:
Proper routes and view components are created properly 🙌

Reply:
@mateo951 Thank you for the approval